### ISSUE TITLE
source-mixpanel: import to Estuary python CDK

### ISSUE THREAD

**Issue by williamhbaker** [estuary/connectors#1426]

**Comment by williamhbaker** [comment]
This is currently noted as being blocked because the API is a paid feature. @Luishfs do you know anything more about that?

**Comment by Luishfs** [comment]
Yes, when running tests (initial snapshot) on this connector on a recent created account, i receive a error claiming i need a paid account to actually use those endpoints.

[They claim](https://mixpanel.com/pricing/) that **Raw data export API** is free, but formatted is not, and it seems the connector runs on Formatted
